channel.py

```diff
@@ -215,6 +217,19 @@ def remove(self):
             self._pending_stream_up = None
         self._gui_channels.remove(self)
 
+    async def is_drops_enable(self,broadcast_id: str) -> bool:
+        response: JsonType = await self._twitch.gql_request(
+            GQL_OPERATIONS["ChannelDrops"].with_variables({"channelID": broadcast_id})
+        )
+        try:
+            available_drops: list | None = response["data"]["channel"]["viewerDropCampaigns"]
+        except Exception:
+            return False
+        if available_drops and len(available_drops) > 0:
```

Comment:
Merely checking for drops existence is quite useless here. Proper querying, considering and tracking of this information is already planned in the rewrite, tracked by #220.